### PR TITLE
Define codeowners for microfrontend definitions

### PR DESCRIPTION
**Pull request by kwiatekus**
**Description**

Changes proposed in this pull request:

- define the only contributors of the MF definitions as code-owners. code owners of the charts usually don't know about the micro front-ends and how they build the UI navigation. Changes in those files should not be merged unnoticed and unreviewed by the owners.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -388,3 +382,26 @@ milv.config.yaml @michal-hudy @m00g3n @aerfio @pPrecel @magicmatatjahu @kazydek
 
 # performance tests
 /tests/perf/ @a-thaler @hisarbalik @lilitgh @suleymanakbas91 @clebs @rakesh-garimella
+
+# The `api-gateway ui` micro front-end definition
+/resources/api-gateway/templates/api-rules-microfrontend.yaml @dariadomagala @parostatkiem @akucharska @Wawrzyn321 @kwiatekus
+
+# add-on ui microfrontend definitions
+/resources/helm-broker/charts/addons-ui/templates/addons-clustermicrofrontend.yaml @dariadomagala @parostatkiem @akucharska @Wawrzyn321 @kwiatekus
+/resources/helm-broker/charts/addons-ui/templates/addons-microfrontend.yaml @dariadomagala @parostatkiem @akucharska @Wawrzyn321 @kwiatekus
```
Comment (akucharska) [inline CODEOWNERS:391]: Should we add also files with images? For example /resources/helm-broker/values.yaml

Comment (sjanota) [reply CODEOWNERS:391]: We should extract it to sub-chart so ownership is clear

Comment (kwiatekus) [reply CODEOWNERS]: done

**Pull request merged**